org.flightgear.FlightGear.yaml

```diff
@@ -104,8 +104,9 @@ modules:
   buildsystem: simple
   sources:
   - type: archive
-    url: https://iweb.dl.sourceforge.net/project/flightgear/release-2020.1/FlightGear-2020.1.3-data.tar.bz2
-    sha256: 2196f4ffcdeaae3a8b0a2b6ebf62a456bfd0198abb6c0619f763204d89804a03
+    archive-type: tar-bzip2
+    url: https://downloads.sourceforge.net/project/flightgear/release-2020.3/FlightGear-2020.3.2-data.tar.bz2
+    md5: a84eb69a9099f4f1e82fadffcf4fa2ee
```

Comment:
md5 is unsafe, please use the sha256 instead. See https://en.wikipedia.org/wiki/MD5#Security for more information.